lib/crds/redirects.rb

```diff
@@ -15,13 +15,28 @@ def initialize
         limit: 1000
       }
     }
+
+    @flex_page_options = {
+      query: {
+        access_token: ENV['CONTENTFUL_ACCESS_TOKEN'],
+        content_type: 'flexPage',
+        limit: 1000
```

Comment:
1 draw back here well have to revisit this logic after 1000 entries have been created, but it should work for now

Reply:
Increased to 2000. Not sure what the actual limitations of this will be but we can always adjust down the road.

Reply:
contentful limit is 1000

essentially what I think we would have to do here is check the total number of entries & paginate records & batch them together with multiple contentful fetches